src/content/en/updates/posts/2016/09/foreign-fetch.md

```diff
@@ -0,0 +1,240 @@
+
+### HTTPS
+
+As with all service worker deployments, the web server you use for serving both your resources and your service worker script [needs to be accessed via HTTPS](https://www.chromium.org/Home/chromium-security/prefer-secure-origins-for-powerful-new-features). Additionally, foreign fetch interception only applies to requests that originate from pages hosted on secure origins, so the clients of your service need to use HTTPS to take advantage of your foreign fetch implementation.
+
+## Using Foreign Fetch
+
+With the prerequisites out of the way, let's dive into the technical details needed to get a foreign fetch service worker up and running.
+
+### Registering your service worker
+
+The first challenge that you're likely to bump into is how to register your service worker. If you've worked with service workers before, you're probably familiar with the following:
+
+{% highlight javascript %}
+// You can't do this!
+if ('serviceWorker' in navigator) {
+  navigator.serviceWorker.register('service-worker.js');
+}
+{% endhighlight %}
+
+This JavaScript code for a first-party service worker registration makes sense in the context of a web app, triggered by a user navigating to a URL you control. But it's not a viable approach to registering a third-party service worker, when the only interaction browser will have with your server is requesting a specific subresource, not a full navigation. If the browser requests, say, an image from a CDN server that you maintain, you can't prepend that snippet of JavaScript to your response and expect that it will be run. A different method of service worker registration, outside the normal JavaScript execution context, is required.
+
+The solution comes in the form of an [HTTP header](https://w3c.github.io/ServiceWorker/spec/service_worker/index.html#link-type-serviceworker) that your server can include in any response:
+
+{% highlight html %}
+Link: </service-worker.js>; rel="serviceworker"; scope="/"
+{% endhighlight %}
+
+Let's break down that example header into its components, each of which is separated by a `;` character.
+
+*   `</service-worker.js>` is required, and is used to specify the path to your service worker file (replace `/service-worker.js` with the appropriate path to your script). This corresponds directly to the [`scriptURL` string](https://developer.mozilla.org/en-US/docs/Web/API/ServiceWorkerContainer/register#Parameters) that would otherwise be passed as the first parameter to <code>navigator.serviceWorker.register()</code>. The value needs to be enclosed in <code><></code> characters (as required by the [`Link` header specification](https://tools.ietf.org/html/rfc5988#section-5)), and if a relative rather than absolute URL is provided, it will be interpreted as being [relative to the location of the response](https://tools.ietf.org/html/rfc3986#section-5.1.3).
```

Comment:
Looks okay to me? Maybe because it's wrapped in the `<code>`.

![image](https://cloud.githubusercontent.com/assets/1749548/18487729/5679f168-79c4-11e6-882a-8951cf3353c6.png)